src/module-vsbridge-indexer-catalog/Plugin/Controller/Catalog/Adminhtml/Product/Action/AttributeSavePlugin.php

```diff
@@ -0,0 +1,85 @@
+<?php declare(strict_types=1);
+/**
+ * @package Divante\VsbridgeIndexerCatalog
+ * @author Marek Mularczyk <mmularczyk@divante.pl>
+ * @copyright 2019 Divante Sp. z o.o.
+ * @license See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Vsf\Indexer\Plugin\Controller\Catalog\Adminhtml\Product\Action;
+
```

Comment:
wrong namespace, please use 
`Divante\VsbridgeIndexerCatalog` instead of `Vsf\Indexer`

Reply:
fixed

Reply:
fixed